avago9361: Handle extra option for some parameters

narasimhan-v opened the pull request:
There are some parameters which need extra options, as
described in the github issue:
avocado-framework-tests/avocado-misc-tests/issues/1776

Handling that via this commit

Reported-by: Gene-Lo
Signed-off-by: Narasimhan V <sim@linux.vnet.ibm.com>

Gene-Lo commented:
After run avago9361.py in Rhel8.1 with Avocado(version:80.0), it still shows " FAIL: Failed to set to off for usefdeonlyencrypt ".

《SOL-log》
![SOL-log](https://user-images.githubusercontent.com/63998174/86886929-5e1d7f00-c12a-11ea-997c-79334152154f.png)

《JOB-log》
[20200708_avago9361_JOB-LOG.zip](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4888959/20200708_avago9361_JOB-LOG.zip)

After we check JOB-log, below cmd are still embedded in the Avocado script:
/root/storcli64 /c0 set copyback=off
/root/storcli64 /c0 set copyback=on
/root/storcli64 /c0 set eghs=off
/root/storcli64 /c0 set eghs=on
/root/storcli64 /c0 show supportssdpatrolread


【Configuration】
[Rhel8.1 Kernel]
4.18.0-147.8.1.el8_1.ppc64le

[FW configuration]
BMC: op940.15.mih-2-0-ga72988e66
PNOR: OP9-v2.4-4.49-prod

[HW configuration]
CPU DD2.3 20 core *2
DIMM (Micron_MTA18ASF2G72PZ-2G9E1) 16G *16
Seagate Jaguar 400GB *2
PSU ACBEL 2000w *2
Broadcom (LSI) MegaRAID 9361-8i SAS3 Controller *1 @ slot10
Seagate Skybolt(HDD; 600GB; 2.5"; 10k; SAS; 12Gb/s; 4Kn/512e; SED) *7

narasimhan-v commented:
> After run avago9361.py in Rhel8.1 with Avocado(version:80.0), it still shows " FAIL: Failed to set to off for usefdeonlyencrypt ".
> 
> 《SOL-log》
> ![SOL-log](https://user-images.githubusercontent.com/63998174/86886929-5e1d7f00-c12a-11ea-997c-79334152154f.png)
> 
Did you check with the vendor, as discussed on the issue previously ?
https://github.com/avocado-framework-tests/avocado-misc-tests/issues/1776#issuecomment-645306611

Gene-Lo commented:
@narasimhan-v 
Last time we ask IBM partners of US team to confirm these cmd, and replied in #1776 (comment)
![image](https://user-images.githubusercontent.com/63998174/86889979-367ce580-c12f-11ea-9e27-239a1a794579.png)
Please kindly check. Many thanks!!

narasimhan-v commented:
> @narasimhan-v
> Last time we ask IBM partners of US team to confirm these cmd, and replied in #1776 (comment)
@Gene-Lo I am not able to find any information related to `usefdeonlyencrypt` in that comment. That is the reason it is not handled in the testscript. Can you please confirm if it needs more handling or if it is an actual bug ?

harish-24 commented:
@narasimhan-v job log please

narasimhan-v commented:
> @narasimhan-v job log please

I don't have access to the adapter, but will print the command instead of running. Will that be fine @harish-24 ?

harish-24 commented:
> > @narasimhan-v job log please
> 
> I don't have access to the adapter, but will print the command instead of running. Will that be fine @harish-24 ?

Makes sense.

narasimhan-v commented:
[job_log.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4951696/job.log)

```
# avocado run avago9361.py -m avago9361.py.data/avago9361.yaml
/usr/local/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:281: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : 331ad58f0bab09aa936dd6ec8282ac5d1eedb1d3
JOB LOG    : /root/avocado/job-results/job-2020-07-21T11.51-331ad58/job.log
 (01/83) avago9361.py:Avago9361.test;rebuildvalue-0-f7f9: PASS (0.00 s)
 (02/83) avago9361.py:Avago9361.test;rebuildvalue-10-98f7: PASS (0.00 s)
 (03/83) avago9361.py:Avago9361.test;rebuildvalue-30-608f: PASS (0.00 s)
 (04/83) avago9361.py:Avago9361.test;rebuildvalue-60-5a61: PASS (0.00 s)
 (05/83) avago9361.py:Avago9361.test;rebuildvalue-100-3daf: PASS (0.00 s)
 (06/83) avago9361.py:Avago9361.test;migvaluevalue-0-7672: PASS (0.00 s)
 (07/83) avago9361.py:Avago9361.test;migvaluevalue-10-e13f: PASS (0.00 s)
 (08/83) avago9361.py:Avago9361.test;migvaluevalue-30-caf7: PASS (0.00 s)
 (09/83) avago9361.py:Avago9361.test;migvaluevalue-60-b09e: PASS (0.00 s)
 (10/83) avago9361.py:Avago9361.test;migvaluevalue-100-7e78: PASS (0.00 s)
 (11/83) avago9361.py:Avago9361.test;ccrate-0-49af: PASS (0.00 s)
 (12/83) avago9361.py:Avago9361.test;ccrate-10-a9e5: PASS (0.00 s)
 (13/83) avago9361.py:Avago9361.test;ccrate-30-d73f: PASS (0.00 s)
 (14/83) avago9361.py:Avago9361.test;ccrate-60-270b: PASS (0.00 s)
 (15/83) avago9361.py:Avago9361.test;ccrate-100-130b: PASS (0.00 s)
 (16/83) avago9361.py:Avago9361.test;bgirate-0-8712: PASS (0.00 s)
 (17/83) avago9361.py:Avago9361.test;bgirate-10-08ee: PASS (0.00 s)
 (18/83) avago9361.py:Avago9361.test;bgirate-30-b572: PASS (0.00 s)
 (19/83) avago9361.py:Avago9361.test;bgirate-60-d2e6: PASS (0.00 s)
 (20/83) avago9361.py:Avago9361.test;bgirate-100-4d8a: PASS (0.00 s)
 (21/83) avago9361.py:Avago9361.test;prrate-0-d97f: PASS (0.00 s)
 (22/83) avago9361.py:Avago9361.test;prrate-10-83e2: PASS (0.00 s)
 (23/83) avago9361.py:Avago9361.test;prrate-30-e1b5: PASS (0.00 s)
 (24/83) avago9361.py:Avago9361.test;prrate-60-b9ff: PASS (0.00 s)
 (25/83) avago9361.py:Avago9361.test;prrate-100-b00b: PASS (0.00 s)
 (26/83) avago9361.py:Avago9361.test;restorehotspare-off-adf2: PASS (0.00 s)
 (27/83) avago9361.py:Avago9361.test;restorehotspare-on-ee2f: PASS (0.00 s)
 (28/83) avago9361.py:Avago9361.test;autorebuild-off-23cc: PASS (0.00 s)
 (29/83) avago9361.py:Avago9361.test;autorebuild-on-b108: PASS (0.00 s)
 (30/83) avago9361.py:Avago9361.test;copyback-off-9c05: PASS (0.00 s)
 (31/83) avago9361.py:Avago9361.test;copyback-on-bc8f: PASS (0.00 s)
 (32/83) avago9361.py:Avago9361.test;eghs-off-e9a6: PASS (0.00 s)
 (33/83) avago9361.py:Avago9361.test;eghs-on-2be9: PASS (0.00 s)
 (34/83) avago9361.py:Avago9361.test;alarm-off-6ac1: PASS (0.00 s)
 (35/83) avago9361.py:Avago9361.test;alarm-on-d28d: PASS (0.00 s)
 (36/83) avago9361.py:Avago9361.test;foreignautoimport-off-17a4: PASS (0.00 s)
 (37/83) avago9361.py:Avago9361.test;foreignautoimport-on-cbd4: PASS (0.00 s)
 (38/83) avago9361.py:Avago9361.test;maintainpdfailhistory-off-9c1d: PASS (0.00 s)
 (39/83) avago9361.py:Avago9361.test;maintainpdfailhistory-on-f5f5: PASS (0.00 s)
 (40/83) avago9361.py:Avago9361.test;ocr-off-161f: PASS (0.00 s)
 (41/83) avago9361.py:Avago9361.test;ocr-on-ee88: PASS (0.00 s)
 (42/83) avago9361.py:Avago9361.test;immediateio-off-b141: PASS (0.00 s)
 (43/83) avago9361.py:Avago9361.test;immediateio-on-f585: PASS (0.00 s)
 (44/83) avago9361.py:Avago9361.test;largeQD-off-0d5e: PASS (0.00 s)
 (45/83) avago9361.py:Avago9361.test;largeQD-on-7c18: PASS (0.00 s)
 (46/83) avago9361.py:Avago9361.test;driveactivityled-off-b723: PASS (0.00 s)
 (47/83) avago9361.py:Avago9361.test;driveactivityled-on-85b7: PASS (0.00 s)
 (48/83) avago9361.py:Avago9361.test;flushwriteverify-off-7208: PASS (0.00 s)
 (49/83) avago9361.py:Avago9361.test;flushwriteverify-on-4f44: PASS (0.00 s)
 (50/83) avago9361.py:Avago9361.test;limitMaxRateSATA-off-a6af: PASS (0.00 s)
 (51/83) avago9361.py:Avago9361.test;limitMaxRateSATA-on-c84a: PASS (0.00 s)
 (52/83) avago9361.py:Avago9361.test;supportssdpatrolread-off-368c: PASS (0.00 s)
 (53/83) avago9361.py:Avago9361.test;supportssdpatrolread-on-7673: PASS (0.00 s)
 (54/83) avago9361.py:Avago9361.test;sgpioforce-off-bf99: PASS (0.00 s)
 (55/83) avago9361.py:Avago9361.test;sgpioforce-on-bee8: PASS (0.00 s)
 (56/83) avago9361.py:Avago9361.test;dpm-off-566a: PASS (0.00 s)
 (57/83) avago9361.py:Avago9361.test;dpm-on-d47a: PASS (0.00 s)
 (58/83) avago9361.py:Avago9361.test;loadbalancemode-off-e880: PASS (0.00 s)
 (59/83) avago9361.py:Avago9361.test;loadbalancemode-on-c72c: PASS (0.00 s)
 (60/83) avago9361.py:Avago9361.test;directpdmapping-off-dad5: PASS (0.00 s)
 (61/83) avago9361.py:Avago9361.test;directpdmapping-on-0400: PASS (0.00 s)
 (62/83) avago9361.py:Avago9361.test;configautobalance-off-ce34: PASS (0.00 s)
 (63/83) avago9361.py:Avago9361.test;configautobalance-on-2312: PASS (0.00 s)
 (64/83) avago9361.py:Avago9361.test;ncq-off-cb24: PASS (0.00 s)
 (65/83) avago9361.py:Avago9361.test;ncq-on-3574: PASS (0.00 s)
 (66/83) avago9361.py:Avago9361.test;abortcconerror-off-63d0: PASS (0.00 s)
 (67/83) avago9361.py:Avago9361.test;abortcconerror-on-55aa: PASS (0.00 s)
 (68/83) avago9361.py:Avago9361.test;batterywarning-off-8783: PASS (0.00 s)
 (69/83) avago9361.py:Avago9361.test;batterywarning-on-ffa4: PASS (0.00 s)
 (70/83) avago9361.py:Avago9361.test;prcorrectunconfiguredareas-off-6571: PASS (0.00 s)
 (71/83) avago9361.py:Avago9361.test;prcorrectunconfiguredareas-on-01af: PASS (0.00 s)
 (72/83) avago9361.py:Avago9361.test;usefdeonlyencrypt-off-9ebd: PASS (0.00 s)
 (73/83) avago9361.py:Avago9361.test;usefdeonlyencrypt-on-07c5: PASS (0.00 s)
 (74/83) avago9361.py:Avago9361.test;cachebypass-off-1d40: PASS (0.00 s)
 (75/83) avago9361.py:Avago9361.test;cachebypass-on-110f: PASS (0.00 s)
 (76/83) avago9361.py:Avago9361.test;activityforlocate-off-7c77: PASS (0.00 s)
 (77/83) avago9361.py:Avago9361.test;activityforlocate-on-7bdc: PASS (0.00 s)
 (78/83) avago9361.py:Avago9361.test;bootwithpinnedcache-off-97f3: PASS (0.00 s)
 (79/83) avago9361.py:Avago9361.test;bootwithpinnedcache-on-0319: PASS (0.00 s)
 (80/83) avago9361.py:Avago9361.test;sesmonitoring-off-26cc: PASS (0.00 s)
 (81/83) avago9361.py:Avago9361.test;sesmonitoring-on-f582: PASS (0.00 s)
 (82/83) avago9361.py:Avago9361.test;failpdonsmarterror-off-3cc5: PASS (0.00 s)
 (83/83) avago9361.py:Avago9361.test;failpdonsmarterror-on-3fd8: PASS (0.00 s)
RESULTS    : PASS 83 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 4.48 s
```

narasimhan-v commented:
> > > @narasimhan-v job log please
> > 
> > 
> > I don't have access to the adapter, but will print the command instead of running. Will that be fine @harish-24 ?
> 
> Makes sense.

Thanks @harish-24 . Done.

harish-24 commented:
@narasimhan-v Looks good, will wait for @Gene-Lo comments/tests.

narasimhan-v commented:
@Gene-Lo Can you please try this patch and let me know if it works ?

Gene-Lo commented:
@harish-24 @narasimhan-v 
I just re-installed Avocado (Version: 80.0), and then run avago9361.py.
However, the result is as same as before:
![image](https://user-images.githubusercontent.com/63998174/88030057-5b307e80-cb6d-11ea-92eb-399ca24cee35.png)

It seems that the new patch hasn't been embedded in the latest Avocado package.
Could you give me the download link of the patch, and illustrate how to install it?
Or when will the patch be embedded in the Avocado package?

narasimhan-v commented:
> @harish-24 @narasimhan-v
> I just re-installed Avocado (Version: 80.0), and then run avago9361.py.
> However, the result is as same as before:
> ![image](https://user-images.githubusercontent.com/63998174/88030057-5b307e80-cb6d-11ea-92eb-399ca24cee35.png)
> 
> It seems that the new patch hasn't been embedded in the latest Avocado package.
> Could you give me the download link of the patch, and illustrate how to install it?
> Or when will the patch be embedded in the Avocado package?

@Gene-Lo 
1. What version of python are you using ?
2. Patch download:
Go to avocado-misc-tests folder and run the below commands.
`wget https://patch-diff.githubusercontent.com/raw/avocado-framework-tests/avocado-misc-tests/pull/1778.patch`
`git apply 1778.patch`
`cd io/disk/Avago_storage_adapter`
Then run your test as you did

Gene-Lo commented:
@narasimhan-v @harish-24 

After we re-test with the new patch, there are too many items FAIL:
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
JOB ID     : 56dc3fcde5a4de08107a468c42e872f4ff135b9d
JOB LOG    : /root/tests/results/job-2020-07-22T10.33-56dc3fc/job.log
 (01/83) avago9361.py:Avago9361.test;rebuildvalue-0-f7f9: FAIL: Failed to show rebuildvalue (167.48 s)
 (02/83) avago9361.py:Avago9361.test;rebuildvalue-10-98f7: FAIL: Failed to show rebuildvalue (165.85 s)
 (03/83) avago9361.py:Avago9361.test;rebuildvalue-30-608f: FAIL: Failed to show rebuildvalue (163.77 s)
 (04/83) avago9361.py:Avago9361.test;rebuildvalue-60-5a61: FAIL: Failed to show rebuildvalue (164.83 s)
 (05/83) avago9361.py:Avago9361.test;rebuildvalue-100-3daf: FAIL: Failed to show rebuildvalue (164.86 s)
 (06/83) avago9361.py:Avago9361.test;migvaluevalue-0-7672: FAIL: Failed to show migvaluevalue (165.85 s)
 (07/83) avago9361.py:Avago9361.test;migvaluevalue-10-e13f: FAIL: Failed to show migvaluevalue (165.34 s)
 (08/83) avago9361.py:Avago9361.test;migvaluevalue-30-caf7: FAIL: Failed to show migvaluevalue (182.14 s)
 (09/83) avago9361.py:Avago9361.test;migvaluevalue-60-b09e: FAIL: Failed to show migvaluevalue (163.09 s)
 (10/83) avago9361.py:Avago9361.test;migvaluevalue-100-7e78: FAIL: Failed to show migvaluevalue (167.83 s)
 (11/83) avago9361.py:Avago9361.test;ccrate-0-49af: PASS (2.97 s)
 (12/83) avago9361.py:Avago9361.test;ccrate-10-a9e5: PASS (3.12 s)
 (13/83) avago9361.py:Avago9361.test;ccrate-30-d73f: PASS (3.09 s)
 (14/83) avago9361.py:Avago9361.test;ccrate-60-270b: PASS (2.96 s)
 (15/83) avago9361.py:Avago9361.test;ccrate-100-130b: PASS (2.96 s)
 (16/83) avago9361.py:Avago9361.test;bgirate-0-8712: PASS (2.97 s)
 (17/83) avago9361.py:Avago9361.test;bgirate-10-08ee: PASS (2.92 s)
 (18/83) avago9361.py:Avago9361.test;bgirate-30-b572: PASS (3.05 s)
 (19/83) avago9361.py:Avago9361.test;bgirate-60-d2e6: PASS (2.98 s)
 (20/83) avago9361.py:Avago9361.test;bgirate-100-4d8a: PASS (2.99 s)
 (21/83) avago9361.py:Avago9361.test;prrate-0-d97f: PASS (2.93 s)
 (22/83) avago9361.py:Avago9361.test;prrate-10-83e2: PASS (3.06 s)
 (23/83) avago9361.py:Avago9361.test;prrate-30-e1b5: PASS (3.02 s)
 (24/83) avago9361.py:Avago9361.test;prrate-60-b9ff: PASS (3.15 s)
 (25/83) avago9361.py:Avago9361.test;prrate-100-b00b: PASS (3.03 s)
 (26/83) avago9361.py:Avago9361.test;restorehotspare-off-adf2: FAIL: Failed to set restorehotspare as False (165.97 s)
 (27/83) avago9361.py:Avago9361.test;restorehotspare-on-ee2f: FAIL: Failed to set restorehotspare as True (117.37 s)
 (28/83) avago9361.py:Avago9361.test;autorebuild-off-23cc: FAIL: Failed to set autorebuild as False (115.98 s)
 (29/83) avago9361.py:Avago9361.test;autorebuild-on-b108: FAIL: Failed to set autorebuild as True (109.01 s)
 (30/83) avago9361.py:Avago9361.test;copyback-off-9c05: FAIL: Failed to set copyback as False (118.16 s)
 (31/83) avago9361.py:Avago9361.test;copyback-on-bc8f: FAIL: Failed to set copyback as True (117.67 s)
 (32/83) avago9361.py:Avago9361.test;eghs-off-e9a6: FAIL: Failed to set eghs as False (109.40 s)
 (33/83) avago9361.py:Avago9361.test;eghs-on-2be9: FAIL: Failed to set eghs as True (114.98 s)
 (34/83) avago9361.py:Avago9361.test;alarm-off-6ac1: FAIL: Failed to set alarm as False (114.71 s)
 (35/83) avago9361.py:Avago9361.test;alarm-on-d28d: FAIL: Failed to set alarm as True (115.28 s)
 (36/83) avago9361.py:Avago9361.test;foreignautoimport-off-17a4: FAIL: Failed to set foreignautoimport as False (117.88 s)
 (37/83) avago9361.py:Avago9361.test;foreignautoimport-on-cbd4: FAIL: Failed to set foreignautoimport as True (121.57 s)
 (38/83) avago9361.py:Avago9361.test;maintainpdfailhistory-off-9c1d: FAIL: Failed to set maintainpdfailhistory as False (114.56 s)
 (39/83) avago9361.py:Avago9361.test;maintainpdfailhistory-on-f5f5: FAIL: Failed to set maintainpdfailhistory as True (108.81 s)
 (40/83) avago9361.py:Avago9361.test;ocr-off-161f: FAIL: Failed to set ocr as False (118.61 s)
 (41/83) avago9361.py:Avago9361.test;ocr-on-ee88: FAIL: Failed to set ocr as True (117.16 s)
 (42/83) avago9361.py:Avago9361.test;immediateio-off-b141: FAIL: Failed to set immediateio as False (116.38 s)
 (43/83) avago9361.py:Avago9361.test;immediateio-on-f585: FAIL: Failed to set immediateio as True (115.72 s)
 (44/83) avago9361.py:Avago9361.test;largeQD-off-0d5e: FAIL: Failed to set largeQD as False (116.61 s)
 (45/83) avago9361.py:Avago9361.test;largeQD-on-7c18: FAIL: Failed to set largeQD as True (110.01 s)
 (46/83) avago9361.py:Avago9361.test;driveactivityled-off-b723: FAIL: Failed to set driveactivityled as False (109.09 s)
 (47/83) avago9361.py:Avago9361.test;driveactivityled-on-85b7: FAIL: Failed to set driveactivityled as True (108.55 s)
 (48/83) avago9361.py:Avago9361.test;flushwriteverify-off-7208: FAIL: Failed to set flushwriteverify as False (114.88 s)
 (49/83) avago9361.py:Avago9361.test;flushwriteverify-on-4f44: FAIL: Failed to set flushwriteverify as True (118.29 s)
 (50/83) avago9361.py:Avago9361.test;limitMaxRateSATA-off-a6af: FAIL: Failed to set limitMaxRateSATA as False (109.34 s)
 (51/83) avago9361.py:Avago9361.test;limitMaxRateSATA-on-c84a: FAIL: Failed to set limitMaxRateSATA as True (115.39 s)
 (52/83) avago9361.py:Avago9361.test;supportssdpatrolread-off-368c: FAIL: Failed to set supportssdpatrolread as False (114.67 s)
 (53/83) avago9361.py:Avago9361.test;supportssdpatrolread-on-7673: FAIL: Failed to set supportssdpatrolread as True (110.89 s)
 (54/83) avago9361.py:Avago9361.test;sgpioforce-off-bf99: FAIL: Failed to set sgpioforce as False (115.50 s)
 (55/83) avago9361.py:Avago9361.test;sgpioforce-on-bee8: FAIL: Failed to set sgpioforce as True (115.94 s)
 (56/83) avago9361.py:Avago9361.test;dpm-off-566a: FAIL: Failed to set dpm as False (109.28 s)
 (57/83) avago9361.py:Avago9361.test;dpm-on-d47a: FAIL: Failed to set dpm as True (118.45 s)
 (58/83) avago9361.py:Avago9361.test;loadbalancemode-off-e880: FAIL: Failed to set loadbalancemode as False (109.40 s)
 (59/83) avago9361.py:Avago9361.test;loadbalancemode-on-c72c: FAIL: Failed to set loadbalancemode as True (110.84 s)
 (60/83) avago9361.py:Avago9361.test;directpdmapping-off-dad5: FAIL: Failed to set directpdmapping as False (109.91 s)
 (61/83) avago9361.py:Avago9361.test;directpdmapping-on-0400: FAIL: Failed to set directpdmapping as True (109.71 s)
 (62/83) avago9361.py:Avago9361.test;configautobalance-off-ce34: FAIL: Failed to set configautobalance as False (110.34 s)
 (63/83) avago9361.py:Avago9361.test;configautobalance-on-2312: FAIL: Failed to set configautobalance as True (116.14 s)
 (64/83) avago9361.py:Avago9361.test;ncq-off-cb24: FAIL: Failed to set ncq as False (109.96 s)
 (65/83) avago9361.py:Avago9361.test;ncq-on-3574: FAIL: Failed to set ncq as True (119.78 s)
 (66/83) avago9361.py:Avago9361.test;abortcconerror-off-63d0: FAIL: Failed to set abortcconerror as False (120.29 s)
 (67/83) avago9361.py:Avago9361.test;abortcconerror-on-55aa: FAIL: Failed to set abortcconerror as True (118.50 s)
 (68/83) avago9361.py:Avago9361.test;batterywarning-off-8783: FAIL: Failed to set batterywarning as False (117.27 s)
 (69/83) avago9361.py:Avago9361.test;batterywarning-on-ffa4: FAIL: Failed to set batterywarning as True (118.95 s)
 (70/83) avago9361.py:Avago9361.test;prcorrectunconfiguredareas-off-6571: FAIL: Failed to set prcorrectunconfiguredareas as False (112.65 s)
 (71/83) avago9361.py:Avago9361.test;prcorrectunconfiguredareas-on-01af: FAIL: Failed to set prcorrectunconfiguredareas as True (110.67 s)
 (72/83) avago9361.py:Avago9361.test;usefdeonlyencrypt-off-9ebd: FAIL: Failed to set usefdeonlyencrypt as False (111.47 s)
 (73/83) avago9361.py:Avago9361.test;usefdeonlyencrypt-on-07c5: FAIL: Failed to set usefdeonlyencrypt as True (109.66 s)
 (74/83) avago9361.py:Avago9361.test;cachebypass-off-1d40: FAIL: Failed to set cachebypass as False (110.21 s)
 (75/83) avago9361.py:Avago9361.test;cachebypass-on-110f: FAIL: Failed to set cachebypass as True (116.31 s)
 (76/83) avago9361.py:Avago9361.test;activityforlocate-off-7c77: FAIL: Failed to set activityforlocate as False (119.90 s)
 (77/83) avago9361.py:Avago9361.test;activityforlocate-on-7bdc: FAIL: Failed to set activityforlocate as True (112.93 s)
 (78/83) avago9361.py:Avago9361.test;bootwithpinnedcache-off-97f3: FAIL: Failed to set bootwithpinnedcache as False (118.35 s)
 (79/83) avago9361.py:Avago9361.test;bootwithpinnedcache-on-0319: FAIL: Failed to set bootwithpinnedcache as True (110.60 s)
 (80/83) avago9361.py:Avago9361.test;sesmonitoring-off-26cc: FAIL: Failed to set sesmonitoring as False (119.13 s)
 (81/83) avago9361.py:Avago9361.test;sesmonitoring-on-f582: FAIL: Failed to set sesmonitoring as True (118.96 s)
 (82/83) avago9361.py:Avago9361.test;failpdonsmarterror-off-3cc5: FAIL: Failed to set failpdonsmarterror as False (116.11 s)
 (83/83) avago9361.py:Avago9361.test;failpdonsmarterror-on-3fd8: FAIL: Failed to set failpdonsmarterror as True (116.09 s)
RESULTS    : PASS 15 | ERROR 0 | FAIL 68 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/tests/results/job-2020-07-22T10.33-56dc3fc/results.html
JOB TIME   : 8411.03 s
↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑


After we check in job.log, most "/root/storcli64 /c0 ~" commands are not supported like below: 
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
INFO | Running '/root/storcli64 /c0 show rebuildvalue'
INFO | Command '/root/storcli64 /c0 show rebuildvalue' finished with 255 after 0.0028684139251708984s
DEBUG| [stdout]
DEBUG| [stdout] syntax error, unexpected TOKEN_UNKNOWN, expecting $end
DEBUG| [stdout]
DEBUG| [stdout]      Storage Command Line Tool  Ver 007.1316.0000.0000 Mar 12, 2020
DEBUG| [stdout]
DEBUG| [stdout]      (c)Copyright 2020, Broadcom Inc. All Rights Reserved.
DEBUG| [stdout]
DEBUG| [stdout]
DEBUG| [stdout] help - lists all the commands with their usage. E.g. storcli help
DEBUG| [stdout] <command> help - gives details about a particular command. E.g. storcli add help
DEBUG| [stdout]
DEBUG| [stdout] List of commands:
DEBUG| [stdout]
DEBUG| [stdout] Commands   Description
DEBUG| [stdout] -------------------------------------------------------------------
DEBUG| [stdout] add        Adds/creates a new element to controller like VD,Spare..etc
DEBUG| [stdout] delete     Deletes an element like VD,Spare
DEBUG| [stdout] show       Displays information about an element
DEBUG| [stdout] set        Set a particular value to a property
DEBUG| [stdout] get        Get a particular value to a property
DEBUG| [stdout] compare    Compares particular value to a property
DEBUG| [stdout] start      Start background operation
DEBUG| [stdout] stop       Stop background operation
DEBUG| [stdout] pause      Pause background operation
DEBUG| [stdout] resume     Resume background operation
DEBUG| [stdout] download   Downloads file to given device
DEBUG| [stdout] expand     expands size of given drive
DEBUG| [stdout] insert     inserts new drive for missing
DEBUG| [stdout] transform  downgrades the controller
DEBUG| [stdout] reset      resets the controller phy
DEBUG| [stdout] split      splits the logical drive mirror
DEBUG| [stdout] /cx        Controller specific commands
DEBUG| [stdout] /ex        Enclosure specific commands
DEBUG| [stdout] /sx        Slot/PD specific commands
DEBUG| [stdout] /vx        Virtual drive specific commands
DEBUG| [stdout] /dx        Disk group specific commands
DEBUG| [stdout] /fall      Foreign configuration specific commands
DEBUG| [stdout] /px        Phy specific commands
DEBUG| [stdout] /lnx       Lane specific commands
DEBUG| [stdout] /[bbu|cv]  Battery Backup Unit, Cachevault commands
DEBUG| [stdout] Other aliases : cachecade, freespace, sysinfo
DEBUG| [stdout]
DEBUG| [stdout] Use a combination of commands to filter the output of help further.
DEBUG| [stdout] E.g. 'storcli cx show help' displays all the show operations on cx.
DEBUG| [stdout] Use verbose for detailed description E.g. 'storcli add  verbose help'
DEBUG| [stdout] Use 'page[=x]' as the last option in all the commands to set the page break.
DEBUG| [stdout] X=lines per page. E.g. 'storcli help page=10'
DEBUG| [stdout] Use J as the last option to print the command output in JSON format
DEBUG| [stdout] Command options must be entered in the same order as displayed in the help of
DEBUG| [stdout] the respective commands.
DEBUG| [stdout] Use 'nolog' option to disable debug logging. E.g. 'storcli show nolog'
DEBUG| [stdout]
↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑

 
Here are job.log and SOL-log:
[job.log & SOL-log.zip](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4958074/job.log.SOL-log.zip)


Here is the configuration of our SUT, and please confirm that we use the same configuration to test:
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
[Rhel8.1 Kernel]
4.18.0-147.el8.ppc64le

[FW config]
BMC: op940.15.mih-2-0-ga72988e66
PNOR: OP9-v2.4-4.49-prod

[HW config]
CPU DD2.3 20core *2
Micron (MTA18ASF2G72PZ-2G9E1) 16G *16
Seagate Jaguar 400GB *1
PSU ACBEL 2000w *2
Broadcom (LSI) MegaRAID 9361-8i SAS3 Controller w/ 8 internal ports *1 @ slot10
Seagate Skybolt; HDD; 600GB; 2.5"; 10k; SAS; 12Gb/s; 4Kn/512e; SED *8
↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑


In addition, we use python3 to install Avocado. Here are the commands of how we install Avocado in Rhel8.1:
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
curl https://avocado-project.org/data/repos/avocado-el.repo -o /etc/yum.repos.d/avocado.repo
yum install python36-devel xz-devel numactl -y
cp /usr/bin/python3 /usr/bin/python
cd /root
git clone https://github.com/open-power-host-os/tests 
cd tests
python avocado-setup.py  --bootstrap
↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑


Many thanks for your kindly support!!

narasimhan-v commented:
@Gene-Lo Thanks for the testing.
There were 2 issues to be fixed, and have fixed them and pushed latest changes.
The issues were:
1. show command did not support rebuild and migrate. So, made changes in yaml to ignore that for those 2 parameters.
2. on and off values were being read as True and False, fixed that by treating it as string in yaml.

This should work for you.


To test the latest changes:
1. go to avocado-misc-tests folder
2. remove the old patch file  1778.patch
3. Revert old changes by the below commands
    `git checkout -- io/disk/Avago_storage_adapter/avago9361.py`
    `git checkout -- io/disk/Avago_storage_adapter/avago9361.py.data/avago9361.yaml`
4. Now get the latest patch by:
    `wget https://patch-diff.githubusercontent.com/raw/avocado-framework-tests/avocado-misc-tests/pull/1778.patch
git apply 1778.patch`
5. Apply the patch:
    `git apply 1778.patch`
6. Run the test as previously

Gene-Lo commented:
@narasimhan-v @harish-24 

After we follow your commands to re-install 1778.patch, some test items become PASS:
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
# avocado run avago9361.py -m avago9361.py.data/avago9361.yaml
/usr/local/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:281: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp.FutureWarning)
JOB ID     : c37c10e2167596ffb28ea2abb143f0fab129f6e6
JOB LOG    : /root/tests/results/job-2020-07-22T15.32-c37c10e/job.log
 (01/83) avago9361.py:Avago9361.test;run-params-value_params-rebuildvalue-valFAIL: Failed to set rebuildvalue as 0 (163.32 s)
 (02/83) avago9361.py:Avago9361.test;run-params-value_params-rebuildvalue-value-10-0265: FAIL: Failed to set rebuildvalue as 10 (150.19 s)
 (03/83) avago9361.py:Avago9361.test;run-params-value_params-rebuildvalue-value-30-a18c: FAIL: Failed to set rebuildvalue as 30 (149.05 s)
 (04/83) avago9361.py:Avago9361.test;run-params-value_params-rebuildvalue-value-60-5d24: FAIL: Failed to set rebuildvalue as 60 (151.19 s)
 (05/83) avago9361.py:Avago9361.test;run-params-value_params-rebuildvalue-value-100-2a0b: FAIL: Failed to set rebuildvalue as 100 (149.71 s)
 (06/83) avago9361.py:Avago9361.test;run-params-value_params-migvaluevalue-value-0-fd9c: FAIL: Failed to set migvaluevalue as 0 (150.52 s)
 (07/83) avago9361.py:Avago9361.test;run-params-value_params-migvaluevalue-value-10-cafe: FAIL: Failed to set migvaluevalue as 10 (149.58 s)
 (08/83) avago9361.py:Avago9361.test;run-params-value_params-migvaluevalue-value-30-02b2: FAIL: Failed to set migvaluevalue as 30 (149.39 s)
 (09/83) avago9361.py:Avago9361.test;run-params-value_params-migvaluevalue-value-60-de6e: FAIL: Failed to set migvaluevalue as 60 (150.04 s)
 (10/83) avago9361.py:Avago9361.test;run-params-value_params-migvaluevalue-value-100-0b87: FAIL: Failed to set migvaluevalue as 100 (149.24 s)
 (11/83) avago9361.py:Avago9361.test;run-params-value_params-ccrate-value-0-49af: PASS (3.01 s)
 (12/83) avago9361.py:Avago9361.test;run-params-value_params-ccrate-value-10-a9e5: PASS (3.14 s)
 (13/83) avago9361.py:Avago9361.test;run-params-value_params-ccrate-value-30-d73f: PASS (3.18 s)
 (14/83) avago9361.py:Avago9361.test;run-params-value_params-ccrate-value-60-270b: PASS (3.12 s)
 (15/83) avago9361.py:Avago9361.test;run-params-value_params-ccrate-value-100-130b: PASS (2.95 s)
 (16/83) avago9361.py:Avago9361.test;run-params-value_params-bgirate-value-0-8712: PASS (3.02 s)
 (17/83) avago9361.py:Avago9361.test;run-params-value_params-bgirate-value-10-08ee: PASS (3.00 s)
 (18/83) avago9361.py:Avago9361.test;run-params-value_params-bgirate-value-30-b572: PASS (3.07 s)
 (19/83) avago9361.py:Avago9361.test;run-params-value_params-bgirate-value-60-d2e6: PASS (3.05 s)
 (20/83) avago9361.py:Avago9361.test;run-params-value_params-bgirate-value-100-4d8a: PASS (3.02 s)
 (21/83) avago9361.py:Avago9361.test;run-params-value_params-prrate-value-0-d97f: PASS (2.98 s)
 (22/83) avago9361.py:Avago9361.test;run-params-value_params-prrate-value-10-83e2: PASS (3.11 s)
 (23/83) avago9361.py:Avago9361.test;run-params-value_params-prrate-value-30-e1b5: PASS (3.11 s)
 (24/83) avago9361.py:Avago9361.test;run-params-value_params-prrate-value-60-b9ff: PASS (3.25 s)
 (25/83) avago9361.py:Avago9361.test;run-params-value_params-prrate-value-100-b00b: PASS (3.05 s)
 (26/83) avago9361.py:Avago9361.test;run-params-state_params-restorehotspare-value-off-2c4d: PASS (3.08 s)
 (27/83) avago9361.py:Avago9361.test;run-params-state_params-restorehotspare-value-on-0d57: PASS (3.15 s)
 (28/83) avago9361.py:Avago9361.test;run-params-state_params-autorebuild-value-off-b2bb: PASS (3.17 s)
 (29/83) avago9361.py:Avago9361.test;run-params-state_params-autorebuild-value-on-ef64: PASS (3.00 s)
 (30/83) avago9361.py:Avago9361.test;run-params-state_params-copyback-value-off-d753: PASS (3.14 s)
 (31/83) avago9361.py:Avago9361.test;run-params-state_params-copyback-value-on-ec22: PASS (2.92 s)
 (32/83) avago9361.py:Avago9361.test;run-params-state_params-eghs-value-off-b6c9: PASS (3.00 s)
 (33/83) avago9361.py:Avago9361.test;run-params-state_params-eghs-value-on-2499: PASS (3.03 s)
 (34/83) avago9361.py:Avago9361.test;run-params-state_params-alarm-value-off-bdd0: PASS (3.02 s)
 (35/83) avago9361.py:Avago9361.test;run-params-state_params-alarm-value-on-cc76: PASS (3.03 s)
 (36/83) avago9361.py:Avago9361.test;run-params-state_params-foreignautoimport-value-off-44b4: PASS (3.03 s)
 (37/83) avago9361.py:Avago9361.test;run-params-state_params-foreignautoimport-value-on-e130: PASS (3.06 s)
 (38/83) avago9361.py:Avago9361.test;run-params-state_params-maintainpdfailhistory-value-off-e9cb: PASS (3.08 s)
 (39/83) avago9361.py:Avago9361.test;run-params-state_params-maintainpdfailhistory-value-on-3f79: PASS (2.95 s)
 (40/83) avago9361.py:Avago9361.test;run-params-state_params-ocr-value-off-87e9: PASS (2.97 s)
 (41/83) avago9361.py:Avago9361.test;run-params-state_params-ocr-value-on-8192: PASS (3.09 s)
 (42/83) avago9361.py:Avago9361.test;run-params-state_params-immediateio-value-off-8745: PASS (3.03 s)
 (43/83) avago9361.py:Avago9361.test;run-params-state_params-immediateio-value-on-c6db: PASS (3.04 s)
 (44/83) avago9361.py:Avago9361.test;run-params-state_params-largeQD-value-off-2a9d: FAIL: Failed to set largeQD as off (151.67 s)
 (45/83) avago9361.py:Avago9361.test;run-params-state_params-largeQD-value-on-1cde: FAIL: Failed to set largeQD as on (156.29 s)
 (46/83) avago9361.py:Avago9361.test;run-params-state_params-driveactivityled-value-off-cddc: PASS (3.00 s)
 (47/83) avago9361.py:Avago9361.test;run-params-state_params-driveactivityled-value-on-14c3: PASS (3.11 s)
 (48/83) avago9361.py:Avago9361.test;run-params-state_params-flushwriteverify-value-off-7589: PASS (3.10 s)
 (49/83) avago9361.py:Avago9361.test;run-params-state_params-flushwriteverify-value-on-066e: PASS (2.99 s)
 (50/83) avago9361.py:Avago9361.test;run-params-state_params-limitMaxRateSATA-value-off-6492: FAIL: Failed to set limitMaxRateSATA as off (162.05 s)
 (51/83) avago9361.py:Avago9361.test;run-params-state_params-limitMaxRateSATA-value-on-26a1: FAIL: Failed to set limitMaxRateSATA as on (154.74 s)
 (52/83) avago9361.py:Avago9361.test;run-params-state_params-supportssdpatrolread-value-off-73a5: PASS (3.14 s)
 (53/83) avago9361.py:Avago9361.test;run-params-state_params-supportssdpatrolread-value-on-ce83: PASS (2.92 s)
 (54/83) avago9361.py:Avago9361.test;run-params-state_params-sgpioforce-value-off-26d0: PASS (2.88 s)
 (55/83) avago9361.py:Avago9361.test;run-params-state_params-sgpioforce-value-on-6425: PASS (2.95 s)
 (56/83) avago9361.py:Avago9361.test;run-params-state_params-dpm-value-off-82cf: FAIL: Failed to set dpm as off (152.19 s)
 (57/83) avago9361.py:Avago9361.test;run-params-state_params-dpm-value-on-f182: FAIL: Failed to set dpm as on (153.61 s)
 (58/83) avago9361.py:Avago9361.test;run-params-state_params-loadbalancemode-value-off-d53c: PASS (3.02 s)
 (59/83) avago9361.py:Avago9361.test;run-params-state_params-loadbalancemode-value-on-d3d0: PASS (2.97 s)
 (60/83) avago9361.py:Avago9361.test;run-params-state_params-directpdmapping-value-off-7a8b: PASS (2.99 s)
 (61/83) avago9361.py:Avago9361.test;run-params-state_params-directpdmapping-value-on-b3ab: PASS (3.20 s)
 (62/83) avago9361.py:Avago9361.test;run-params-state_params-configautobalance-value-off-5215: FAIL: Failed to set configautobalance as off (153.08 s)
 (63/83) avago9361.py:Avago9361.test;run-params-state_params-configautobalance-value-on-f24c: FAIL: Failed to set configautobalance as on (154.12 s)
 (64/83) avago9361.py:Avago9361.test;run-params-state_params-ncq-value-off-7225: PASS (3.15 s)
 (65/83) avago9361.py:Avago9361.test;run-params-state_params-ncq-value-on-185d: PASS (2.96 s)
 (66/83) avago9361.py:Avago9361.test;run-params-state_params-abortcconerror-value-off-ad5e: PASS (3.05 s)
 (67/83) avago9361.py:Avago9361.test;run-params-state_params-abortcconerror-value-on-3157: PASS (3.03 s)
 (68/83) avago9361.py:Avago9361.test;run-params-state_params-batterywarning-value-off-4554: PASS (3.06 s)
 (69/83) avago9361.py:Avago9361.test;run-params-state_params-batterywarning-value-on-d757: PASS (3.07 s)
 (70/83) avago9361.py:Avago9361.test;run-params-state_params-prcorrectunconfiguredareas-value-off-d993: PASS (2.93 s)
 (71/83) avago9361.py:Avago9361.test;run-params-state_params-prcorrectunconfiguredareas-value-on-c341: PASS (2.97 s)
 (72/83) avago9361.py:Avago9361.test;run-params-state_params-usefdeonlyencrypt-value-off-86e9: FAIL: Failed to set usefdeonlyencrypt as off (152.13 s)
 (73/83) avago9361.py:Avago9361.test;run-params-state_params-usefdeonlyencrypt-value-on-0ef8: PASS (3.08 s)
 (74/83) avago9361.py:Avago9361.test;run-params-state_params-cachebypass-value-off-459a: PASS (3.02 s)
 (75/83) avago9361.py:Avago9361.test;run-params-state_params-cachebypass-value-on-b16b: PASS (2.98 s)
 (76/83) avago9361.py:Avago9361.test;run-params-state_params-activityforlocate-value-off-45fd: PASS (3.14 s)
 (77/83) avago9361.py:Avago9361.test;run-params-state_params-activityforlocate-value-on-4b31: PASS (2.99 s)
 (78/83) avago9361.py:Avago9361.test;run-params-state_params-bootwithpinnedcache-value-off-60a2: PASS (3.15 s)
 (79/83) avago9361.py:Avago9361.test;run-params-state_params-bootwithpinnedcache-value-on-c7e8: PASS (3.02 s)
 (80/83) avago9361.py:Avago9361.test;run-params-state_params-sesmonitoring-value-off-1c77: PASS (3.04 s)
 (81/83) avago9361.py:Avago9361.test;run-params-state_params-sesmonitoring-value-on-2268: PASS (3.01 s)
 (82/83) avago9361.py:Avago9361.test;run-params-state_params-failpdonsmarterror-value-off-ec6d: PASS (3.03 s)
 (83/83) avago9361.py:Avago9361.test;run-params-state_params-failpdonsmarterror-value-on-f098: PASS (3.13 s)
RESULTS    : PASS 64 | ERROR 0 | FAIL 19 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/tests/results/job-2020-07-22T15.32-c37c10e/results.html
JOB TIME   : 3101.09 s
↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑


Here is the actual commands of how we re-installed 1778.patch:
(We removed 1778.patch first, and then input below commands to re-install it)
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
[root@mihawk-sut5 avocado-misc-tests]# pwd
/root/tests/tests/avocado-misc-tests
[root@mihawk-sut5 avocado-misc-tests]# #(Check if 1778.patch has been removed)
[root@mihawk-sut5 avocado-misc-tests]# updatedb
[root@mihawk-sut5 avocado-misc-tests]# locate 1778.patch
[root@mihawk-sut5 avocado-misc-tests]# locate *.patch
/root/tests/tests/avocado-misc-tests/cpu/dwh.py.data/fofd.patch
/root/tests/tests/avocado-misc-tests/cpu/ebizzy.py.data/Fix-build-issues-with-ebizzy.patch
/root/tests/tests/avocado-misc-tests/cpu/em_smt_folding_test.py.data/Fix-build-issues-with-ebizzy.patch
/root/tests/tests/avocado-misc-tests/cpu/linsched.py.data/fix.patch
/root/tests/tests/avocado-misc-tests/fuzz/fsfuzzer.py.data/fsfuzz_fix.patch
/root/tests/tests/avocado-misc-tests/generic/interbench.py.data/makefile_fix.patch
/root/tests/tests/avocado-misc-tests/io/disk/dbench.py.data/dbench_startup.patch
/root/tests/tests/avocado-misc-tests/io/disk/iozone.py.data/makefile.patch
/root/tests/tests/avocado-misc-tests/kernel/tlbflush.py.data/tlbflush.patch
/root/tests/tests/avocado-misc-tests/memory/eatmemory.py.data/eatmem_getch.patch
/root/tests/tests/avocado-misc-tests/memory/libhugetlbfs.py.data/elflink.patch
/root/tests/tests/avocado-misc-tests/memory/ndctl_selftest.py.data/ppc.patch
/root/tests/tests/avocado-misc-tests/perf/blogbench.py.data/config_guess.patch
/root/tests/tests/avocado-misc-tests/perf/lmbench.py.data/0001-Fix-build-issues-with-lmbench.patch
/root/tests/tests/avocado-misc-tests/perf/lmbench.py.data/0002-Changing-shebangs-on-lmbench-scripts.patch
/root/tests/tests/avocado-misc-tests/perf/lmbench.py.data/fix_add_os_type.patch
/root/tests/tests/avocado-misc-tests/perf/lmbench.py.data/makefile.patch
/root/tests/tests/avocado-misc-tests/perf/unixbench.py.data/make1.patch
/root/tests/tests/avocado-misc-tests/toolchain/ltrace.py.data/ltrace.patch
/usr/share/doc/python3-pycurl/tests/matrix/curl-7.19.0-sslv2-2b0e09b0f98.patch
/usr/share/doc/python3-pycurl/tests/matrix/curl-7.19.0-sslv2-c66b0b32fba-modified.patch
/usr/share/doc/python3-pycurl/tests/matrix/openssl-1.0.1e-fix_pod_syntax-1.patch
[root@mihawk-sut5 avocado-misc-tests]#
[root@mihawk-sut5 avocado-misc-tests]#
[root@mihawk-sut5 avocado-misc-tests]#
[root@mihawk-sut5 avocado-misc-tests]# git checkout -- io/disk/Avago_storage_adapter/avago9361.py
[root@mihawk-sut5 avocado-misc-tests]# git checkout -- io/disk/Avago_storage_adapter/avago9361.py.data/avago9361.yaml
[root@mihawk-sut5 avocado-misc-tests]# wget https://patch-diff.githubusercontent.com/raw/avocado-framework-tests/avocado-misc-tests/pull/1778.patch git apply 1778.patch
--2020-07-22 15:22:38--  https://patch-diff.githubusercontent.com/raw/avocado-framework-tests/avocado-misc-tests/pull/1778.patch
Resolving patch-diff.githubusercontent.com (patch-diff.githubusercontent.com)... 140.82.114.4
Connecting to patch-diff.githubusercontent.com (patch-diff.githubusercontent.com)|140.82.114.4|:443... connected.
HTTP request sent, awaiting response... 200 OK
Cookie coming from patch-diff.githubusercontent.com attempted to set domain to github.com
Length: unspecified [text/plain]
Saving to: ‘1778.patch’

1778.patch                           [ <=>                                                      ]   7.85K  --.-KB/s    in 0s

2020-07-22 15:22:44 (52.4 MB/s) - ‘1778.patch’ saved [8037]

--2020-07-22 15:22:44--  http://git/
Resolving git (git)... failed: Name or service not known.
wget: unable to resolve host address ‘git’
--2020-07-22 15:22:44--  http://apply/
Resolving apply (apply)... failed: Name or service not known.
wget: unable to resolve host address ‘apply’
--2020-07-22 15:22:45--  http://1778.patch/
Resolving 1778.patch (1778.patch)... failed: Name or service not known.
wget: unable to resolve host address ‘1778.patch’
FINISHED --2020-07-22 15:22:45--
Total wall clock time: 6.5s
Downloaded: 1 files, 7.8K in 0s (52.4 MB/s)
[root@mihawk-sut5 avocado-misc-tests]# ls
1778.patch  cpu  fs  fuzz  generic  io  kernel  LICENSE  memory  perf  ras  README.rst  requirements-travis.txt  toolchain
[root@mihawk-sut5 avocado-misc-tests]# git apply 1778.patch
[root@mihawk-sut5 avocado-misc-tests]#
↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑


Here are the job.log and Manual-Test log of avago9361.py, while the result of Manual-Test log is as same as the result of  job.log:
[job.log & Manual-Test log.zip](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4959079/job.log.Manual-Test.log.zip)


Here is the configuration of our SUT:
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
[Rhel8.1 Kernel]
4.18.0-147.el8.ppc64le

[FW config]
BMC: op940.15.mih-2-0-ga72988e66
PNOR: OP9-v2.4-4.49-prod

[HW config]
CPU DD2.3 20core *2
Micron (MTA18ASF2G72PZ-2G9E1) 16G *16
Seagate Jaguar 400GB *1
PSU ACBEL 2000w *2
Broadcom (LSI) MegaRAID 9361-8i SAS3 Controller w/ 8 internal ports *1 @ slot10
Seagate Skybolt; HDD; 600GB; 2.5"; 10k; SAS; 12Gb/s; 4Kn/512e; SED *8
↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑

Please kindly check. Many thanks!!

narasimhan-v commented:
@Gene-Lo Thanks for this report.
We seem to be closing in on finalising the script.

Issues:
1. rebuildvalue - It is actually changed into rebuildvalue. Changed in yaml now
2. migvaluevalue - That is a typo, though there is no migrate feature that can be set on / off. There are 2 new parameters added, which are reconrate and eccbucketleakrate. I have added both here as well now.
3. largeQD - Not supported in your controller. Fixed in code.
4. limitMaxRateSATA - Not supported in your controller. Fixed in code.
5. dpm - Not supported in your controller. Fixed in code.
6. configautobalance - Not supported in your controller. Fixed in code.
7. usefdeonlyencrypt - This looks to be a real bug, as setting it to on PASSes. Please check with vendor.

Request to test again, as previously.

Gene-Lo commented:
@narasimhan-v @harish-24 

Today we removed and then re-installed 1778.patch, in order to test avago9361.py again. However, all test-items show "ERROR: 'int' object has no attribute 'stdout_text'", which makes the result FAIL:

↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
( ...... SKIP ...... )
 (80/88) avago9361.py:Avago9361.test;run-params-state_params-cachebypass-value-on-b16b: ERROR: 'int' object has no attribute 'stdout_text' (1.48 s)
 (81/88) avago9361.py:Avago9361.test;run-params-state_params-activityforlocate-value-off-45fd: ERROR: 'int' object has no attribute 'stdout_text' (1.51 s)
 (82/88) avago9361.py:Avago9361.test;run-params-state_params-activityforlocate-value-on-4b31: ERROR: 'int' object has no attribute 'stdout_text' (1.46 s)
 (83/88) avago9361.py:Avago9361.test;run-params-state_params-bootwithpinnedcache-value-off-60a2: ERROR: 'int' object has no attribute 'stdout_text' (1.47 s)
 (84/88) avago9361.py:Avago9361.test;run-params-state_params-bootwithpinnedcache-value-on-c7e8: ERROR: 'int' object has no attribute 'stdout_text' (1.38 s)
 (85/88) avago9361.py:Avago9361.test;run-params-state_params-sesmonitoring-value-off-1c77: ERROR: 'int' object has no attribute 'stdout_text' (1.39 s)
 (86/88) avago9361.py:Avago9361.test;run-params-state_params-sesmonitoring-value-on-2268: ERROR: 'int' object has no attribute 'stdout_text' (1.33 s)
 (87/88) avago9361.py:Avago9361.test;run-params-state_params-failpdonsmarterror-value-off-ec6d: ERROR: 'int' object has no attribute 'stdout_text' (1.44 s)
 (88/88) avago9361.py:Avago9361.test;run-params-state_params-failpdonsmarterror-value-on-f098: ERROR: 'int' object has no attribute 'stdout_text' (1.45 s)
RESULTS    : PASS 0 | ERROR 88 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/tests/results/job-2020-07-23T10.58-81a9254/results.html
JOB TIME   : 129.53 s
↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑

Here is the SOL-log and job.log:
[SOL-log & job.log.zip](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4963733/SOL-log.job.log.zip)

Please kindly fix the bug. Many thanks!!

narasimhan-v commented:
@Gene-Lo Sorry, that is my mistake. Fixed it in the latest code. Please try.

Gene-Lo commented:
@narasimhan-v @harish-24 

Fortunately, after we re-test avago9361.py with the latest patch, it remains only 2 items FAIL:
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
JOB ID     : 413d9c40edf148a5894d64892fae6de6a36ae33d
JOB LOG    : /root/tests/results/job-2020-07-23T12.55-413d9c4/job.log
 (01/88) avago9361.py:Avago9361.test;run-params-value_params-rebuildrate-value-0-4e9e: PASS (3.03 s)
 (02/88) avago9361.py:Avago9361.test;run-params-value_params-rebuildrate-value-10-6568: PASS (3.04 s)
 (03/88) avago9361.py:Avago9361.test;run-params-value_params-rebuildrate-value-30-c862: PASS (3.04 s)
 (04/88) avago9361.py:Avago9361.test;run-params-value_params-rebuildrate-value-60-9a9d: PASS (2.97 s)
 (05/88) avago9361.py:Avago9361.test;run-params-value_params-rebuildrate-value-100-23b3: PASS (2.97 s)
 (06/88) avago9361.py:Avago9361.test;run-params-value_params-reconrate-value-0-7f82: PASS (3.08 s)
 (07/88) avago9361.py:Avago9361.test;run-params-value_params-reconrate-value-10-3c29:  [root@mihawk-sut5 ~]# tail -f screenlog.0
 (01/88) avago9361.py:Avago9361.test;run-params-value_params-rebuildrate-value-0-4e9e: PASS (3.03 s)
 (02/88) avago9361.py:Avago9361.test;run-params-value_params-rebuildrate-value-10-6568: PASS (3.04 s)
 (03/88) avago9361.py:Avago9361.test;run-params-value_params-rebuildrate-value-30-c862: PASS (3.04 s)
 (04/88) avago9361.py:Avago9361.test;run-params-value_params-rebuildrate-value-60-9a9d: PASS (2.97 s)
 (05/88) avago9361.py:Avago9361.test;run-params-value_params-rebuildrate-value-100-23b3: PASS (2.97 s)
 (06/88) avago9361.py:Avago9361.test;run-params-value_params-reconrate-value-0-7f82: PASS (3.08 s)
 (07/88) avago9361.py:Avago9361.test;run-params-value_params-reconrate-value-10-3c29: PASS (3.04 s)
 (08/88) avago9361.py:Avago9361.test;run-params-value_params-reconrate-value-30-6216: PASS (3.05 s)
 (09/88) avago9361.py:Avago9361.test;run-params-value_params-reconrate-value-60-2376: PASS (3.14 s)
 (10/88) avago9361.py:Avago9361.test;run-params-value_params-reconrate-value-100-ac25: PASS (3.02 s)
 (11/88) avago9361.py:Avago9361.test;run-params-value_params-eccbucketleakrate-value-0-a264: FAIL: setting eccbucketleakrate as 0 failed (                           110.70 s)
 (12/88) avago9361.py:Avago9361.test;run-params-value_params-eccbucketleakrate-value-10-0096: PASS (3.08 s)
 (13/88) avago9361.py:Avago9361.test;run-params-value_params-eccbucketleakrate-value-30-58a0: PASS (3.10 s)
 (14/88) avago9361.py:Avago9361.test;run-params-value_params-eccbucketleakrate-value-60-8109: PASS (3.14 s)
 (15/88) avago9361.py:Avago9361.test;run-params-value_params-eccbucketleakrate-value-100-2331: PASS (3.05 s)
 (16/88) avago9361.py:Avago9361.test;run-params-value_params-ccrate-value-0-49af: PASS (3.00 s)
 (17/88) avago9361.py:Avago9361.test;run-params-value_params-ccrate-value-10-a9e5: PASS (3.03 s)
 (18/88) avago9361.py:Avago9361.test;run-params-value_params-ccrate-value-30-d73f: PASS (3.05 s)
 (19/88) avago9361.py:Avago9361.test;run-params-value_params-ccrate-value-60-270b: PASS (3.02 s)
 (20/88) avago9361.py:Avago9361.test;run-params-value_params-ccrate-value-100-130b: PASS (3.02 s)
 (21/88) avago9361.py:Avago9361.test;run-params-value_params-bgirate-value-0-8712: PASS (3.04 s)
 (22/88) avago9361.py:Avago9361.test;run-params-value_params-bgirate-value-10-08ee: PASS (3.07 s)
 (23/88) avago9361.py:Avago9361.test;run-params-value_params-bgirate-value-30-b572: PASS (3.02 s)
 (24/88) avago9361.py:Avago9361.test;run-params-value_params-bgirate-value-60-d2e6: PASS (3.09 s)
 (25/88) avago9361.py:Avago9361.test;run-params-value_params-bgirate-value-100-4d8a: PASS (3.01 s)
 (26/88) avago9361.py:Avago9361.test;run-params-value_params-prrate-value-0-d97f: PASS (3.11 s)
 (27/88) avago9361.py:Avago9361.test;run-params-value_params-prrate-value-10-83e2: PASS (3.11 s)
 (28/88) avago9361.py:Avago9361.test;run-params-value_params-prrate-value-30-e1b5: PASS (2.98 s)
 (29/88) avago9361.py:Avago9361.test;run-params-value_params-prrate-value-60-b9ff: PASS (3.08 s)
 (30/88) avago9361.py:Avago9361.test;run-params-value_params-prrate-value-100-b00b: PASS (3.05 s)
 (31/88) avago9361.py:Avago9361.test;run-params-state_params-restorehotspare-value-off-2c4d: PASS (3.12 s)
 (32/88) avago9361.py:Avago9361.test;run-params-state_params-restorehotspare-value-on-0d57: PASS (3.11 s)
 (33/88) avago9361.py:Avago9361.test;run-params-state_params-autorebuild-value-off-b2bb: PASS (3.02 s)
 (34/88) avago9361.py:Avago9361.test;run-params-state_params-autorebuild-value-on-ef64: PASS (3.04 s)
 (35/88) avago9361.py:Avago9361.test;run-params-state_params-copyback-value-off-d753: PASS (3.07 s)
 (36/88) avago9361.py:Avago9361.test;run-params-state_params-copyback-value-on-ec22: PASS (3.06 s)
 (37/88) avago9361.py:Avago9361.test;run-params-state_params-eghs-value-off-b6c9: PASS (3.09 s)
 (38/88) avago9361.py:Avago9361.test;run-params-state_params-eghs-value-on-2499: PASS (3.09 s)
 (39/88) avago9361.py:Avago9361.test;run-params-state_params-alarm-value-off-bdd0: PASS (3.07 s)
 (40/88) avago9361.py:Avago9361.test;run-params-state_params-alarm-value-on-cc76: PASS (3.08 s)
 (41/88) avago9361.py:Avago9361.test;run-params-state_params-foreignautoimport-value-off-44b4: PASS (3.10 s)
 (42/88) avago9361.py:Avago9361.test;run-params-state_params-foreignautoimport-value-on-e130: PASS (3.09 s)
 (43/88) avago9361.py:Avago9361.test;run-params-state_params-maintainpdfailhistory-value-off-e9cb: PASS (3.03 s)
 (44/88) avago9361.py:Avago9361.test;run-params-state_params-maintainpdfailhistory-value-on-3f79: PASS (3.19 s)
 (45/88) avago9361.py:Avago9361.test;run-params-state_params-ocr-value-off-87e9: PASS (3.21 s)
 (46/88) avago9361.py:Avago9361.test;run-params-state_params-ocr-value-on-8192: PASS (3.06 s)
 (47/88) avago9361.py:Avago9361.test;run-params-state_params-immediateio-value-off-8745: PASS (3.11 s)
 (48/88) avago9361.py:Avago9361.test;run-params-state_params-immediateio-value-on-c6db: PASS (3.04 s)
 (49/88) avago9361.py:Avago9361.test;run-params-state_params-largeQD-value-off-2a9d: CANCEL: largeQD not supported (3.09 s)
 (50/88) avago9361.py:Avago9361.test;run-params-state_params-largeQD-value-on-1cde: CANCEL: largeQD not supported (3.04 s)
 (51/88) avago9361.py:Avago9361.test;run-params-state_params-driveactivityled-value-off-cddc: PASS (3.09 s)
 (52/88) avago9361.py:Avago9361.test;run-params-state_params-driveactivityled-value-on-14c3: PASS (3.02 s)
 (53/88) avago9361.py:Avago9361.test;run-params-state_params-flushwriteverify-value-off-7589: PASS (3.08 s)
 (54/88) avago9361.py:Avago9361.test;run-params-state_params-flushwriteverify-value-on-066e: PASS (3.04 s)
 (55/88) avago9361.py:Avago9361.test;run-params-state_params-limitMaxRateSATA-value-off-6492: CANCEL: limitMaxRateSATA not supported (3.11                            s)
 (56/88) avago9361.py:Avago9361.test;run-params-state_params-limitMaxRateSATA-value-on-26a1: CANCEL: limitMaxRateSATA not supported (3.08                            s)
 (57/88) avago9361.py:Avago9361.test;run-params-state_params-supportssdpatrolread-value-off-73a5: PASS (3.06 s)
 (58/88) avago9361.py:Avago9361.test;run-params-state_params-supportssdpatrolread-value-on-ce83: PASS (3.07 s)
 (59/88) avago9361.py:Avago9361.test;run-params-state_params-sgpioforce-value-off-26d0: PASS (3.06 s)
 (60/88) avago9361.py:Avago9361.test;run-params-state_params-sgpioforce-value-on-6425: PASS (3.10 s)
 (61/88) avago9361.py:Avago9361.test;run-params-state_params-dpm-value-off-82cf: CANCEL: dpm not supported (3.09 s)
 (62/88) avago9361.py:Avago9361.test;run-params-state_params-dpm-value-on-f182: CANCEL: dpm not supported (3.06 s)
 (63/88) avago9361.py:Avago9361.test;run-params-state_params-loadbalancemode-value-off-d53c: PASS (3.04 s)
 (64/88) avago9361.py:Avago9361.test;run-params-state_params-loadbalancemode-value-on-d3d0: PASS (3.12 s)
 (65/88) avago9361.py:Avago9361.test;run-params-state_params-directpdmapping-value-off-7a8b: PASS (3.05 s)
 (66/88) avago9361.py:Avago9361.test;run-params-state_params-directpdmapping-value-on-b3ab: PASS (3.12 s)
 (67/88) avago9361.py:Avago9361.test;run-params-state_params-configautobalance-value-off-5215: CANCEL: setting configautobalance as off no                           t supported (3.08 s)
 (68/88) avago9361.py:Avago9361.test;run-params-state_params-configautobalance-value-on-f24c: CANCEL: setting configautobalance as on not                            supported (3.07 s)
 (69/88) avago9361.py:Avago9361.test;run-params-state_params-ncq-value-off-7225: PASS (3.04 s)
 (70/88) avago9361.py:Avago9361.test;run-params-state_params-ncq-value-on-185d: PASS (3.05 s)
 (71/88) avago9361.py:Avago9361.test;run-params-state_params-abortcconerror-value-off-ad5e: PASS (3.07 s)
 (72/88) avago9361.py:Avago9361.test;run-params-state_params-abortcconerror-value-on-3157: PASS (3.07 s)
 (73/88) avago9361.py:Avago9361.test;run-params-state_params-batterywarning-value-off-4554: PASS (3.02 s)
 (74/88) avago9361.py:Avago9361.test;run-params-state_params-batterywarning-value-on-d757: PASS (3.10 s)
 (75/88) avago9361.py:Avago9361.test;run-params-state_params-prcorrectunconfiguredareas-value-off-d993: PASS (3.07 s)
 (76/88) avago9361.py:Avago9361.test;run-params-state_params-prcorrectunconfiguredareas-value-on-c341: PASS (3.05 s)
 (77/88) avago9361.py:Avago9361.test;run-params-state_params-usefdeonlyencrypt-value-off-86e9: FAIL: setting usefdeonlyencrypt as off fail                           ed (112.19 s)
 (78/88) avago9361.py:Avago9361.test;run-params-state_params-usefdeonlyencrypt-value-on-0ef8: PASS (3.04 s)
 (79/88) avago9361.py:Avago9361.test;run-params-state_params-cachebypass-value-off-459a: PASS (3.05 s)
 (80/88) avago9361.py:Avago9361.test;run-params-state_params-cachebypass-value-on-b16b: PASS (3.09 s)
 (81/88) avago9361.py:Avago9361.test;run-params-state_params-activityforlocate-value-off-45fd: PASS (3.09 s)
 (82/88) avago9361.py:Avago9361.test;run-params-state_params-activityforlocate-value-on-4b31: PASS (3.09 s)
 (83/88) avago9361.py:Avago9361.test;run-params-state_params-bootwithpinnedcache-value-off-60a2: PASS (3.05 s)
 (84/88) avago9361.py:Avago9361.test;run-params-state_params-bootwithpinnedcache-value-on-c7e8: PASS (2.99 s)
 (85/88) avago9361.py:Avago9361.test;run-params-state_params-sesmonitoring-value-off-1c77: PASS (3.05 s)
 (86/88) avago9361.py:Avago9361.test;run-params-state_params-sesmonitoring-value-on-2268: PASS (3.06 s)
 (87/88) avago9361.py:Avago9361.test;run-params-state_params-failpdonsmarterror-value-off-ec6d: PASS (3.01 s)
 (88/88) avago9361.py:Avago9361.test;run-params-state_params-failpdonsmarterror-value-on-f098: PASS (3.06 s)
RESULTS    : PASS 78 | ERROR 0 | FAIL 2 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 8
JOB HTML   : /root/tests/results/job-2020-07-23T12.55-413d9c4/results.html
JOB TIME   : 490.07 s
↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑


Here is the configuration of our SUT:
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
[Rhel8.1 Kernel]
4.18.0-147.8.1.el8_1.ppc64le

[FW config]
BMC: op940.15.mih-2-0-ga72988e66
PNOR: OP9-v2.4-4.49-prod

[HW config]
CPU DD2.3 20core *2
Micron (MTA18ASF2G72PZ-2G9E1) 16G *16
SAMSUNG PM983 M.2 *1
PSU ACBEL 2000w *2
Broadcom (LSI) MegaRAID 9361-8i SAS3 Controller w/ 8 internal ports *1 @ slot10
Seagate Skybolt; HDD; 600GB; 2.5"; 10k; SAS; 12Gb/s; 4Kn/512e; SED *8
↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑


Here is the job.log:
[job.zip](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4964136/job.zip)


Regarding to the FAIL items (No.11 & No.77), we tried to manual-test them, but the results still FAIL:
[Manual-Test.zip](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4964142/Manual-Test.zip)

Please kindly check if the FAIL items have a chance to be fixed, or if we should directly ask the vendor.
Many thanks for you kindly support!!

narasimhan-v commented:
@Gene-Lo Thanks.
From a test case perspective, it is working fine.
For those 2 FAIL items, since they are failing manually also, you will have to check with vendor.

narasimhan-v commented:
@Gene-Lo Thanks for testing it with the multiple iterations of fixes. Test is now stable.

narasimhan-v commented:
@harish-24 We are ready with a stable test now, hope it can be merged.
Thanks in advance.
